content/04.discussion.md

```diff
@@ -1,3 +1,25 @@
 ## Discussion
 
-Stub in discussion section   
+Through OpenPBTA, we used an open, reproducible analysis framework within GitHub to bring together researchers and clinicians across the nation to genomically characterize 943 pediatric brain tumors across 59 distinct histologies.
+To our knowledge, this was the first large-scale, collaborative open analysis of genomic data coupled with open manuscript writing in which we comprehensively analyzed the largest cohort of pediatric brain tumors to date.
+We used available WGS, WXS, and RNA-Seq data to generate high-confidence consensus SNV and CNV calls, prioritize putative oncogenic fusions, and create over 40 code- and scientifically-reviewed scalable modules for common downstream cancer genomics analyses which have been used to create new knowledge as well as define new questions.
+We detect and show expected frequencies of genomic lesions, mutational signatures, and aberrantly regulated signaling pathways across multiple pediatric brain tumor histologies.
+Molecular subtyping information was largely not collected nor available for this cohort, and if available (e.g.: sparse medulloblastoma subtypes), it had to be manually curated from pathology reports and/or free text clinical data fields.
+Furthermore, this tumor cohort has yet to be profiled by DNA methylation. 
+Thus, to enable more targeted analyses to interrogate the biology of specific tumor subtypes, we created subtyping modules aligned with WHO molecularly-defined diagnoses.
+By working closely with pathologists and clinicians, we were able to determine a research-grade integrated diagnosis for 44% (1260/2840) of biospecimens and determine mis-diagnosed or mis-identified samples. 
+
+We applied multiple previously validated classifiers to RNA expression data.
+We predicted medulloblastoma subtypes [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263], whether tumors have dysfunctional _TP53_ [@doi:10.1016/j.celrep.2018.03.076], and their degree of telomerase activity [@doi:10.1038/s41467-020-20474-9].
+By assessing _TP53_ and telomerase activity prospectively from expression data, information previously only attainable with genomic sequencing and/or qPCR, we can quickly add knowledge to our existing biological understanding of these tumors.
+
+OpenPBTA has become a foundational layer to which we will continue to add other genomic modalities and analyses, such as germline, methylation, single cell, epigenomic, mRNA splicing, imaging, and model drug response data.
+We created the OpenPBTA project to provide robust code and data to the pediatric oncology community, as well as attract additional collaborators and new analyses to accelerate therapeutic translation for children with cancer, goals we are seeing play out in real-time.
```

Comment:
I think this sentence should be earlier in the discussion with in the first 3 sentences.

Reply:
✅